libnymea-zigbee/backends/deconz/zigbeenetworkdeconz.h

```diff
@@ -68,6 +68,7 @@ class ZigbeeNetworkDeconz : public ZigbeeNetwork
     QString m_protocolVersion;
     QString m_firmwareVersion;
 
+    QList<ZigbeeNetworkReply*> m_requestQueue;
```

Comment:
This could actually be a `QQueue<ZigbeeNetworkReply*>`, for better readability, you can enqueu and dequeue instead of takeFirst, append and such...but not so important...